doc/source/style.rst

```diff
@@ -2,7 +2,7 @@
 Documentation Style
 *******************
 
-PyAnsys packages should pick either the Google or Numpy documentation
+Ansys packages should pick either the Google or Numpy documentation
```

Comment:
No need to change this line. I guess this change was made automatically.

```suggestion
PyAnsys packages should pick either the Google or Numpy documentation
```

Reply:
True story... that one was my mistake. Thanks @jorgepiloto